scripts/cpplint.py

```diff
@@ -4614,7 +4614,7 @@ def Check__CPROVER_(filename, clean_lines, linenum, error):
     error: The function to call with any errors found.
   """
   line = clean_lines.lines[linenum]
-  match = Match(r'.*__CPROVER_.*', line)
+  match = Match(r'.*\"__CPROVER_.*\"', line)
```

Comment:
I would appreciate if the commit message could have a body explaining why this change is right.

Reply:
I made the lint change a separate PR, together with the removal of some NOLINT comments. #5000